src/Physics/Multinucleon/EventGen/MECGenerator.h

```diff
@@ -66,10 +67,18 @@ public :
   // in the kPSTlctl phase space
   double GetXSecMaxTlctl( const Interaction& inter ) const;
 
+  double GetXSecMaxTlctl( const Interaction & inter, const Range1D_t Tl_range, const Range1D_t ctl_range ) const;
```

Comment:
Ranges to be references. You you prefer them to be copies, remove the const, but I think references is better